Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -408,6 +409,8 @@ public class HTTPClient {
         public var redirectConfiguration: RedirectConfiguration
         /// Default client timeout, defaults to no timeouts.
         public var timeout: Timeout
+        /// Timeout of pooled connections
+        public var poolingTimeout: TimeAmount?
```

Comment:
Is this a good name for this setting?

Reply:
I'd make it a bit longer and more descriptive. Maybe `maximumAllowedIdleTimeInConnectionPool` or so? @Lukasa ?

Reply:
Ok, I set it to it for now, thanks.